### PR TITLE
Bump weekly 0.3 32

### PR DESCRIPTION
**Pull request by matthme**
* Moves most of the rust dependencies to the root Cargo.toml
* bumps to holochain-0.3.0-beta-dev.32

### CODE REVIEW

**Comment by ThetaSinner** [comment]
At this version Lair keystore had been bumped to 0.4.0, I think that will need to change in the workspace deps and this should be good to go https://github.com/holochain/holochain/blob/holochain-0.3.0-beta-dev.32/Cargo.lock#L4490

File: Cargo.toml [hunk 1]
```diff
@@ -7,3 +7,26 @@ members = [
 ]
 resolver = "2"
 
+
+[workspace.dependencies]
+# NEW_VERSION Update holochain dependencies here
+holochain = { version = "0.3.0-beta-dev.34", features = ["sqlite-encrypted"] }
```
Comment (ThetaSinner) [inline Cargo.toml]: `sqlite-encrypted` is now the default! :)

File: crates/hc_launch/src-tauri/src/cli.rs [hunk 2]
```diff
@@ -3,23 +3,25 @@
...
-use std::path::{PathBuf, Path};
 
 use crate::launch_tauri::launch_tauri;
 use crate::prepare_webapp;
 use holochain_cli_sandbox::cmds::{Create, Existing, NetworkCmd, NetworkType};
 
-
 #[derive(Debug, Parser)]
-#[command(version = "0.0.14 (holochain 0.3.x)")]
+#[command(version = "0.0.14 (holochain 0.3.0-beta-dev.32)")]
```
Comment (ThetaSinner) [inline crates/hc_launch/src-tauri/src/cli.rs]: .34 now?

File: crates/hc_launch/src-tauri/src/cli.rs [hunk 3]
```diff
@@ -526,24 +557,25 @@ pub fn get_running_ports(path: PathBuf, n_expected: usize) -> Vec<u16> {
...
-            panic!("ERROR: No running snadbox conductors found. If you use the --reuse-conductors flag there need to be existing sandbox conductors running.\n {}", e);
-          },
-          _ => {
-            println!("[hc launch] ERROR: Not enough running sandbox conductors found. If you use the --reuse-conductors flag there need to be as many running sandbox conductors as mentioned in the .hc file.\n {}", e);
-            panic!("ERROR: No running snadbox conductors found. If you use the --reuse-conductors flag there need to be as many running sandbox conductors as mentioned in the .hc file.\n {}", e);
-          }
+      Err(e) => match n {
+        0 => {
+          println!("[hc launch] ERROR: No running sandbox conductors found. If you use the --reuse-conductors flag there need to be existing sandbox conductors running.\n {}", e);
+          panic!("ERROR: No running snadbox conductors found. If you use the --reuse-conductors flag there need to be existing sandbox conductors running.\n {}", e);
```
Comment (ThetaSinner) [inline crates/hc_launch/src-tauri/src/cli.rs]:
```suggestion
          panic!("ERROR: No running sandbox conductors found. If you use the --reuse-conductors flag there need to be existing sandbox conductors running.\n {}", e);
```

File: crates/hc_launch/src-tauri/src/cli.rs [hunk 4]
```diff
@@ -526,24 +557,25 @@ pub fn get_running_ports(path: PathBuf, n_expected: usize) -> Vec<u16> {
...
         }
-      }
+        _ => {
+          println!("[hc launch] ERROR: Not enough running sandbox conductors found. If you use the --reuse-conductors flag there need to be as many running sandbox conductors as mentioned in the .hc file.\n {}", e);
+          panic!("ERROR: No running snadbox conductors found. If you use the --reuse-conductors flag there need to be as many running sandbox conductors as mentioned in the .hc file.\n {}", e);
```
Comment (ThetaSinner) [inline crates/hc_launch/src-tauri/src/cli.rs]:
```suggestion
          panic!("ERROR: No running sandbox conductors found. If you use the --reuse-conductors flag there need to be as many running sandbox conductors as mentioned in the .hc file.\n {}", e);
```

**Review by ThetaSinner — APPROVED** [review]

**Comment by matthme** [comment]
I think this is now good to merge. It's only UI related dependencies that fail in the test and I won't invest time into that since we're not continuing with the tauri launcher UI.

**Pull request merged**